estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,89 @@
+        <field name="name">Estate property form view</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <form string="Tests">
+                <sheet>
+                    <div>
+                        <h1>
+                            <field name = "name"/>
+                        </h1>
+                    </div>
+                    <field name="tag_ids" widget="many2many_tags"/>
+                    <group>
+                        <group>
+                            <field name = "property_type_id"/>
+                            <field name = "postcode"/>
+                            <field name = "date_availability"/>
+                        </group>
+                        <group>
+                            <field name = "expected_price"/>
+                            <field name = "best_price"/>
+                            <field name = "selling_price"/>
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Description">
+                             <group>
+                                <field name="description"/>
+                                <field name="bedrooms"/>
+                                <field name="living_area"/>
+                                <field name="facades"/>
+                                <field name="garage"/>
+                                <field name="garden"/>
+                                <field name="garden_area"/>
+                                <field name="garden_orientation"/>
+                                <field name="total_area"/>
+                             </group>
+                        </page>
+                        <page string="Offers">
+                             <field name="offer_ids"/>
+                        </page>
+                        <page string="Other Info">
+                             <group>
+                                <field name="salesman_id"/>
+                                <field name="buyer_id"/>
+                             </group>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="estate_property_view_search" model="ir.ui.view">
+        <field name="name">Estate property view search</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Tests">
+                <field name="name"/>
+                <field name="postcode"/>
+                <field name="expected_price"/>
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="facades"/>
+                <filter string="Available" name="Available" domain="['|',('state', '=', 'New'),('state', '=', 'Offer Received')]"/>
+            </search>
```

Comment:
the domain is good here, an other option would be to combine these two conditions together:
domain="[('state', 'in', ['New', 'Offer Received'])]"
Which is more readable to me